python/lsst/analysis/tools/interfaces/_interfaces.py

```diff
@@ -43,6 +44,36 @@
 from numpy.typing import NDArray
 
 
+@runtime_checkable
+class Tensor(Protocol):
+    r"""This is an interface only class and is intended to represent data that
+    is 2+ dimensions.
+
+    Technically one could use this for scalars or 1D arrays,
+    but for those the Scalar or Vector interface should be preferred.
+
+    `Tensor`\ s abstract around the idea of a multidimensional array, and work
```

Comment:
What's the purpose of the "\ s"? Is it an attempt to preserve a space? Is it needed? And will it work? I'd imagine you may need to double-escape "\\" here to make this work.

Reply:
In the past this has been how you have a link (to Tensor in this case) and have an s at the end of it. You can't include it in the quotes or sphinx can't find the object. the \ is an escape sequence on the space that says don't include this in formatting.